auth/build.gradle.kts

```diff
@@ -11,6 +11,7 @@ android {
 
     lintOptions {
         disable("UnusedQuantity")
+        disable("UnknownNullness") // TODO fix in future PR
```

Comment:
I wonder if there's a way we could disable the nullness check on classes that have `@RestrictTo` but leave it on other `public` classes?

Reply:
Damn, that's a great idea! I'll file an issue for Tor.

Reply:
https://issuetracker.google.com/issues/113439283